Src/library/wel/support/wel_api.e

```diff
@@ -955,8 +955,41 @@ feature -- API
 			is_class: class
 		end
 
+feature -- Externals DPI
+
+	cwin_get_dpi_for_window (hwnd: POINTER): INTEGER
+				-- Returns the dots per inch (dpi) value for the associated window.
+				-- `hwnd` The window you want to get information about.
+		note
+			EIS:"name=GetDpiForWindow", "src=https://msdn.microsoft.com/en-us/library/windows/desktop/mt748624(v=vs.85).aspx", "protocol=uri"
+		external
+			"C [macro %"wel.h%"] (HWND): EIF_INTEGER"
+		alias
+			"GetDpiForWindow"
+		end
+
+	cwin_mul_div (a_number, a_numerator, a_denominator: INTEGER): INTEGER
```

Comment:
Is this function different from `(a_number.as_integer_64 * a_numerator / a_denominator).rounded`? I'm asking to make sure there is no significant performance degradation due to the external function calls to this function instead of inline arithmetic.